<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>695</width>
    <height>662</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EBSD-GUI</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout" stretch="2">
      <item>
       <layout class="QHBoxLayout" name="topLayout" stretch="1,0,2">
        <item>
         <layout class="QVBoxLayout" name="systemViewerLayout" stretch="0,0,0,0">
          <item>
           <widget class="QLabel" name="folderLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>NO FOLDER OPENED</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTreeView" name="systemViewer">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>320</width>
              <height>320</height>
             </size>
            </property>
            <property name="mouseTracking">
             <bool>false</bool>
            </property>
            <property name="tabletTracking">
             <bool>true</bool>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::CurrentChanged|QAbstractItemView::DoubleClicked|QAbstractItemView::SelectedClicked</set>
            </property>
            <property name="tabKeyNavigation">
             <bool>true</bool>
            </property>
            <property name="dragEnabled">
             <bool>false</bool>
            </property>
            <property name="dragDropOverwriteMode">
             <bool>false</bool>
            </property>
            <property name="alternatingRowColors">
             <bool>false</bool>
            </property>
            <property name="animated">
             <bool>true</bool>
            </property>
            <attribute name="headerStretchLastSection">
             <bool>true</bool>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="consoleLog">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="inputLayout">
            <item>
             <widget class="QLabel" name="consolePrompt">
              <property name="text">
               <string>&gt;&gt;&gt;</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="MplWidget" name="MplWidget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>320</width>
            <height>320</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>695</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_Workfolder"/>
   </widget>
   <widget class="QMenu" name="menuProcessing">
    <property name="title">
     <string>Processing</string>
    </property>
    <addaction name="actionProcessingMenu"/>
    <addaction name="actionROI"/>
    <addaction name="actionPattern_Center"/>
   </widget>
   <widget class="QMenu" name="menuPlot">
    <property name="title">
     <string>Plotting</string>
    </property>
    <addaction name="actionSignalNavigation"/>
   </widget>
   <widget class="QMenu" name="menuDictionary_indexing">
    <property name="title">
     <string>Indexing</string>
    </property>
    <addaction name="actionDictionary_indexing"/>
    <addaction name="actionHough_indexing"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuProcessing"/>
   <addaction name="menuPlot"/>
   <addaction name="menuDictionary_indexing"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_Workfolder">
   <property name="text">
    <string>Open Workfolder...</string>
   </property>
   <property name="statusTip">
    <string comment="LOL">Select a folder containing patterns</string>
   </property>
  </action>
  <action name="actionProcessingMenu">
   <property name="text">
    <string>N/S improvement</string>
   </property>
   <property name="statusTip">
    <string>Perform processing on a pattern</string>
   </property>
  </action>
  <action name="actionSignalNavigation">
   <property name="text">
    <string>Signal navigation</string>
   </property>
  </action>
  <action name="actionDictionary_indexing">
   <property name="text">
    <string>Dictionary indexing</string>
   </property>
  </action>
  <action name="actionPattern_Center">
   <property name="text">
    <string>Pattern center</string>
   </property>
  </action>
  <action name="actionROI">
   <property name="text">
    <string>ROI</string>
   </property>
  </action>
  <action name="actionHough_indexing">
   <property name="text">
    <string>Hough indexing</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MplWidget</class>
   <extends>QWidget</extends>
   <header>mplwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
